examples/basic-globalize-compiler/README.md

```diff
@@ -0,0 +1,26 @@
+# Basic Globalize Compiler example
+
+On this example, we assume you know the Globalize usage basics. So, it focuses on
+the Globalize Compiler and on the Globalize runtime modules.
```

Comment:
Generally we don't really do the hard wrapping, just write this out on the same line. Or is this how you're doing it everywhere in this repo?